src/qrl/core/txs/Transaction.py

```diff
@@ -29,8 +29,16 @@
 class Transaction(object, metaclass=ABCMeta):
     """
     Abstract Base class to be derived by all other transactions
-    """
 
+    Every operation possible on the QRL blockchain is represented by a
+    corresponding Transaction type (until smart contract functionality is
+    introduced). This is similar to how business logic is done in Hyperledger
+    Sawtooth.
```

Comment:
Please stick to the topic. It is not worth to mention if it is similar to Hyperledger Sawtooth.